### PR TITLE
various fixes to chart and driver

### PR DESCRIPTION
**Pull request by Panaetius**
- renames myApp to csi-driver
- makes storage class name/driver name configurable
- gitignores generated Kind template
- deletes Controller secrets that weren't used
- removes sops secrets
- adds helm lint action
- minor fixes in the chart

### CODE REVIEW

File: deploy/csi-rclone/templates/csi-rclone-storageclass.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: {{ include "chart.fullname" . }}
```
Comment (olevski) [inline deploy/csi-rclone/templates/csi-rclone-storageclass.yaml:4]: The storage class name should be hardcoded or in the values file. I see you already have the value for it in the values file.

File: deploy/csi-rclone/values.yaml [hunk 2]
```diff
@@ -1,37 +1,19 @@
+storageClassName: csi-rclone 
 namespace:
   enabled: true
```
Comment (olevski) [inline deploy/csi-rclone/values.yaml:3]: I dont understand why we need this. `helm install` has a `--create-namespace` option that will crate the namespace you are trying to install in if it des not exist. The helm chart should not be responsible for making the namespace - that is up to the admin or whoever is installing the chart.

Comment (olevski) [reply deploy/csi-rclone/values.yaml:3]: But it is ok we can clean this up in a follow up PR. It is just confusing to have the helm chart make this.

File: deploy/chartpress.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    imagePrefix: renku/
+    resetTag: ""
+    resetVersion: 0.1.0
+    repo:
+      git: SwissDataScienceCenter/helm-charts
+      published: https://swissdatasciencecenter.github.io/helm-charts
+    images:
+      csi-rclone:
+        contextPath: ../
+        dockerFilePath: ../Dockerfile
```
Comment (olevski) [inline deploy/chartpress.yaml:12]: I dont see a Dockerfile in this branch at all. Just FYI.

File: pkg/rclone/rclone.go [hunk 4]
```diff
@@ -216,7 +214,13 @@ func (r Rclone) Unmount(ctx context.Context, volumeId string, targetPath string)
 		return err
 	}
 	if resp.StatusCode != 200 {
-		return fmt.Errorf("unmounting failed: couldn't delete mount: %s", string(body))
+		var result map[string]interface{}
+		json.Unmarshal(body, &result)
+		val, ok := result["error"]
+		if !ok || val != "mount not found" {
```
Comment (olevski) [inline pkg/rclone/rclone.go:220]:
```suggestion
		errorMsg, found := result["error"]
		if !found || errorMsg != "mount not found" {
```

Not a big deal but it is nicer if the names were more descriptive.

**Review by olevski — CHANGES_REQUESTED** [review]
Just a few minor things. It looks good.

File: deploy/csi-rclone/values.yaml (hunk 2 above)
Comment (Panaetius) [reply deploy/csi-rclone/values.yaml:3]: I think this is just some leftover from some fork

**Review by olevski — APPROVED** [review]

**Pull request merged**